### PR TITLE
Y2K problems

### PR DESCRIPTION
**Pull request by larsbrinkhoff**
- SCNV writes something like `;8` for 2018.
- TJ6 writes 1918.

### CODE REVIEW

**Comment by larsbrinkhoff** [comment]
I haven't tested this yet.

**Comment by atsampson** [comment]
> We better check all uses of RFDATE (and SFDATE).

We'll need to look at all the calls to FILBLK, RFDATE, RQDATE, SFDATE, SRDATE for #853 anyway...

File: src/tj6/tj6.29 [hunk 1]
```diff
@@ -4443,12 +4443,11 @@ BEGDAT:	.RDATE	B,
...
 	LDB	A,[600,,B]
 	PUSHJ	P,BEG6D
-	MOVE	H,[440700,,[ASCIZ /, 19/]]
+	MOVE	H,[440700,,[ASCIZ /, /]]
 	PUSHJ	P,ASHTK
-	LDB	A,[360600,,B]	;1st year digit
-	PUSHJ	P,BEG6D
-	LDB	A,[300600,,B]
-	PUSHJ	P,BEG6D
+	.RYEAR	A,
```
[the remark this answers is not in the document]
Comment (larsbrinkhoff) [reply src/tj6/tj6.29]: I wanted to do a small univasive change, but you may be right that RQDATE is the best way to go.

**Comment by larsbrinkhoff** [comment]
This begs the question, use 7, 8, or 9 bits of year from RQDATE?  7 is the currently documented number, 8 is likely quite safe, 9 would be nice to avoid a year 2156 problem.  Right, grandchildren?

**Comment by atsampson** [comment]
I'd go with 7 bits for now; we can update it when we decide how to change the date format.

**Comment by larsbrinkhoff** [comment]
Now tested.

**Comment by larsbrinkhoff** [comment]
Thanks for the extra testing!

**Comment by larsbrinkhoff** [comment]
GitLab says the job has passed, but the status hasn't reached GitHub.  Maybe Microsoft snarfed it.

**Comment by eswenson1** [comment]
Oh no! I thought we’d have at least 6 months before Microsoft ruined github!

**Pull request merged**